libraries/src/Document/Renderer/Html/ScriptsRenderer.php

```diff
@@ -186,7 +186,11 @@ private function renderElement($item): string
 
         // Check if script uses media version.
         if ($version && !str_contains($src, '?') && ($mediaVersion || $version !== 'auto')) {
-            $src .= '?' . ($version === 'auto' ? $mediaVersion : $version);
+            $src .= '?' . match($version) {
```

Comment:
```suggestion
            $src .= '?' . match ($version) {
```
Fix PHPCS. See also example on https://www.php.net/manual/en/control-structures.match.php